index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS Documentation</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav id="navbar">
+        <header><h4>JS Documentation</h4></header>
+        <ul>
+            <li><a class="nav-link" href="#introduction">Introduction</a></li>
+            <li><a class="nav-link" href="#What_you_should_already_know">What you should already know</a></li>
+            <li><a class="nav-link" href="#JavaScript_and_Java">JavaScript and Java</a></li>
+            <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+            <li><a class="nav-link" href="#Declaring_variables">Declaring variables</a></li>
+            <li><a class="nav-link" href="#Variable_scope">Variable scope</a></li>
+            
+        </ul>
+
+    </nav>
+    <main id="main-doc">
+        <section id="introduction" class="main-section">
+            <header>
+                <h2>Introduction</h2>
+            </header>
+            <p>JavaScript is a cross-platform, object-oriented scripting language.
+                It is a small and lightweight language. Inside a
+                host environment (for example, a web browser), JavaScript can be connected 
+                to the objects of its environment to provide
+                programmatic control over them.
+            </p>
+
```

Comment:
Kindly remove all blank lines from this section as well as other parts of your code for best HTML practice.